example-functions.php

```diff
@@ -299,8 +299,9 @@ function yourprefix_register_demo_metabox() {
 	$cmb_demo->add_field( array(
 		'name'    => __( 'Test Multi Checkbox', 'cmb2' ),
 		'desc'    => __( 'field description (optional)', 'cmb2' ),
-		'id'      => $prefix . 'multicheckbox',
+		'id'      => $prefix . 'multicheckbox2',
 		'type'    => 'multicheck',
+		'multiple' => true, // Store values in individual rows

```

Comment:
Should comment this out by default since that is not the default behavior and is how optional arguments are noted in this file.